packages/remix-dev/cli/commands.ts

```diff
@@ -251,26 +236,11 @@ export async function generateEntry(
     return;
   }
 
-  let clientRenderer = deps["@remix-run/react"] ? "react" : undefined;
-
-  if (!clientRenderer) {
-    console.error(
-      colors.error(
-        `Could not determine runtime. Please install the following: @remix-run/react`
-      )
-    );
-    return;
-  }
```

Comment:
This didn't seem necessary - we can assume React in Remix apps for now and the foreseeable future